src/ResourceManager/Cdn/Commands.Cdn/Endpoint/SetAzureRmCdnEndpoint.cs

```diff
@@ -27,7 +27,7 @@ namespace Microsoft.Azure.Commands.Cdn.Endpoint
     [Cmdlet(VerbsCommon.Set, "AzureRmCdnEndpoint"), OutputType(typeof(PSEndpoint))]
     public class SetAzureRmCdnEndpoint : AzureCdnCmdletBase
```

Comment:
This should really SupportShouldProcess and have execution guarded by ShouldProcess as well